syncr_backend/network/handle_frontend.py

```diff
@@ -474,19 +475,41 @@ def handle_input_name(
     :param conn: socket.accept() connection
     :return: None
     """
-    if request['drop_name'] is None:
-        response = {
-            'status': 'error',
-            'error': ERR_INVINPUT,
-        }
+    # This code assumes that the user has already
+    # created a folder to initialize as a drop.
+    #
+    # First, we check to see if tracker already contains drop id
+    # If so, do nothing, send message failure back
+    #
+    # Else, initialize directory with 'drop_name' as the name of the drop
+
+    # TODO: Allow user to select directory location from UI (on frontend).
+
+    directory = request['directory']
+    drop_name = os.path.basename(directory)
+
+    status = 'error'
+    result = 'failure'
+
+    # TODO: Change 'False' to check if drop id already exists in tracker
```

Comment:
Also, it would be close to impossible to have conflicting drop ids, so you really don't need to check this.

Reply:
The reason why I have this statement is that, from a frontend perspective,  if the user chooses a directory that's already been initialized, nothing should happen. There's probably another way of going about it which I'll look into

Reply:
Ah, good point.  There are files in ~/.5yncr/drops (you can see how to get this directory in `drop_metadata.list_drops()`) that are named the drop IDs and with the content being where on the filesystem that drop is; you could write something like `list_drops` that returns the drop locations and compare the paths (using some functions is `os.path`, maybe call `os.path.realpath` on both and compare?).  Or you could look to see if there is a `.5yncr` dir in the new drop location.

Reply:
I will be addressing this problem within the next couple of days